spec/components/question/phone_number_component/view_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "rails_helper"
+
+RSpec.describe Question::PhoneNumberComponent::View, type: :component do
+  let(:question_page) { build :page, answer_type: "phone_number" }
+  let(:answer_text) { nil }
+  let(:question) { DataStruct.new(phone_number: answer_text, question_text: question_page.question_text, hint_text: question_page.hint_text, answer_settings: nil) }
+  let(:extra_question_text_suffix) { nil }
+  let(:form_builder) do
+    GOVUKDesignSystemFormBuilder::FormBuilder.new(:form, question,
+                                                  ActionView::Base.new(ActionView::LookupContext.new(nil), {}, nil), {})
+  end
+
+  before do
+    render_inline(described_class.new(form_builder:, question:, extra_question_text_suffix:))
+  end
+
+  describe "when component is phone number field" do
+    it "renders the question text as a heading" do
+      expect(page.find("h1")).to have_text(question.question_text)
+    end
+
+    it "renders a telephone input field" do
+      expect(page).to have_css("input[type='tel'][name='form[phone_number]']")
+    end
+
+    it "renders at one half the width of the parent container" do
+      expect(page.native.to_html).to include('class="govuk-input govuk-!-width-one-half"')
```

Comment:
If you go with the `width` change above:
```suggestion
      expect(page.native.to_html).to include('class="govuk-input govuk-input--width-20"')
```

Reply:
Good suggestions, I've added that suggests in a separate commit.